query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -19,7 +19,7 @@ import prismaQueries from "../bench/queries.json";
 import { run, bench, group, baseline } from "mitata";
 
 import { QueryEngine as WasmBaseline } from "query-engine-wasm-baseline";
-import { QueryEngine as WasmLatest } from "query-engine-wasm-latest";
+import { QueryEngine as WasmLatest } from "query-engine-wasm-latest/postgresql/query_engine";
```

Comment:
Should the loading be dynamic and per provider otherwise we risk executing with the wrong set of instructions?

Reply:
The benchmark only uses Postgres. But I could add a comment for this.

Reply:
(I'm happy to see the first green badges on the CI for `driver-adapters` tests. I can now add the missing comment, push, and leave in peace)